scale-encode/Cargo.toml

```diff
@@ -14,22 +14,25 @@ keywords.workspace = true
 include.workspace = true
 
 [features]
-default = ["derive", "primitive-types", "bits"]
+default = ["std", "derive", "primitive-types", "bits"]
 
-# Include the derive proc macro
-derive = ["dep:scale-encode-derive"]
+# Activates std feature.
+std = ["scale-info/std"]
```

Comment:
```suggestion
std = ["scale-info?/std"]
```

Reply:
scale-info is not optional to my understanding?

Reply:
Ah good point!